tasks/pipeline-instcode.yml

```diff
@@ -42,8 +42,6 @@
       dest: "/usr/lib/archivematica/MCPServer"
     - src: "{{ archivematica_src_dir }}/archivematica/src/MCPServer/share"
       dest: "/usr/share/archivematica/MCPServer"
-    - src: "{{ archivematica_src_dir }}/archivematica/src/MCPServer/share/mysql"
-      dest: "/usr/share/dbconfig-common/data/archivematica-mcp-server/install/mysql"
```

Comment:
I think this directory is only required by dbconfig-common, which was only used by the package installer, not ansible, so it's safe to remove.